### ISSUE TITLE
.scss method with .sourceMaps produces an empty sourcemap file.

### ISSUE THREAD

**Issue by SethTompkins** [JeffreyWay/laravel-mix#449]
- Laravel Mix Version: 0.8.1
- Node Version (`node -v`): 7.6.0 
- NPM Version (`npm -v`): 4.2.0
- OS: macOS 10.12.3

### Description:

`.sass` method with `.sourceMaps()` generates an empty sourcemap

mix config: 

```js
mix.js('resources/assets/js/main.js', 'public/js/bundle.js')
.js('resources/assets/js/index-page.js', 'public/js/index-page.js')
.js('resources/assets/js/icons.js', 'public/js/icons.js')
.extract(['jquery', 'babel-polyfill'])
.autoload({
	jquery: ['$', 'jQuery', 'window.jQuery'],
	'headroom.js': ['Headroom']
})
.sass('resources/assets/sass/style.scss', 'public/css')
.disableNotifications()
.options({
	processCssUrls: false
})
.sourceMaps()
.browserSync({
	proxy: process.env.LOCAL_PROXY,
	files: [
		'app/**/*.php',
	        'resources/views/**/*.php',
		'public/js/**/*.js',
	        'public/css/**/*.css'
	]
})
```

### Steps To Reproduce:

Run `npm run watch` with the provided config.

**Comment by ankurk91** [comment]
Same as #560

**Comment by webmarkelov** [comment]
@SethTompkins  Hi! Could you post your mix & webpack config?

**Comment by SethTompkins** [comment]
Here you go:

```
require('dotenv').config()
const { mix, config } = require('laravel-mix')

mix.js('resources/assets/js/main.js', 'public/js/bundle.js')
.js('resources/assets/js/index-page.js', 'public/js/index-page.js')
.js('resources/assets/js/icons.js', 'public/js/icons.js')
.extract(['jquery', 'babel-polyfill'])
.autoload({
	jquery: ['$', 'jQuery', 'window.jQuery'],
	'headroom.js': ['Headroom']
})
.sass('resources/assets/sass/style.scss', 'public/css')
.disableNotifications()
.options({
	processCssUrls: false
})
.sourceMaps()
.browserSync(process.env.LOCAL_PROXY)
```

**Comment by ankurk91** [comment]
Fixed in #654
